content/_includes/events.macros.njk

```diff
@@ -38,16 +39,50 @@ params:
     {%- if title -%}
       {{- layout.title(title) -}}
     {%- endif -%}
-    <ol {{ 'reversed' if reverse else '' }}>
+    <ol {{ 'reversed' if reverse else '' }} class="event-list">
       {%- for event in events -%}
-        <li>
-          {%- if event | isFuture -%}
-            <strong>[{{ utility.datetime(event.date) }}]</strong>
-          {%- else -%}
-            [{{ utility.datetime(event.date) }}]
-          {%- endif %}
-          {{ utility.link_if(event.venue, event.url) }}{% if event.adr %} {{ '--' | typogr | safe }}
-          {{ event.adr }}{% endif %}
+        {% set event_slug = [event.date, loop.index] | join('-') | slug %}
```

Comment:
This creates really long/weird IDs: `id="mon-oct-26-2020-00:00:00-gmt+0000-(coordinated-universal-time)-1"`